.env.example

```diff
@@ -37,6 +37,7 @@ CONVERSATION_MAX_PARTICIPANTS=50
 CONVERSATION_PRELOAD_COUNT=1000
 JWT_ACCESS_SECRET=token
 JWT_ACCESS_TOKEN_EXPIRES_IN=10800
+ENCRYPTION_MESSAGE_EXPIRED_IN=2592000
```

Comment:
let's use comments in the end 
`ENCRYPTION_MESSAGE_EXPIRED_IN=2592000  # 30 days`